GitUI/Theming/ThemeModule.cs

```diff
@@ -11,6 +11,8 @@ namespace GitUI.Theming
 {
     public static class ThemeModule
     {
+        private static bool _suppressWin32Hooks;
```

Comment:
I'd add documention like `/// To be activated by the TestAccessor only.`

Reply:
renamed variable to indicate test-only scope